src/ConnectedMode.UnitTests/UI/ManageBinding/ManageBindingViewModelTests.cs

```diff
@@ -705,13 +705,26 @@ public async Task DisplayBindStatusAsync_WhenProjectIsBoundButProjectNotFoundOnS
     {
         var sonarCloudConnection = new ServerConnection.SonarCloud("organization", credentials: validCredentials);
         SetupBoundProjectThatDoesNotExistOnServer(sonarCloudConnection);
-        
+
         await testSubject.DisplayBindStatusAsync();
-        
+
         testSubject.SelectedProject.Should().BeNull();
         testSubject.BoundProject.Should().BeNull();
     }
-    
+
+    [TestMethod]
+    public async Task DisplayBindStatusAsync_WhenProjectWasBoundAndBecomesUnbound_UpdatesCurrentProjectAndBindingInfoToNull()
+    {
+        await InitializeBoundProject();
```

Comment:
The two tests are not identical:
The InitializeBoundProject does something very important: initializes the view model propeties (SelectedConnectionInfo, SelectedProject, BoundProject) with objects whereas the other test sets them to null.

I will keep the test DisplayBindStatusAsync_WhenProjectIsNotBound_SelectedConnectionShouldBeEmpty, because it is a regression test, i.e. it will make sure that when we have a bound project and then unbind, the properties in the view model are set to null

Reply:
Removing the `await InitializeBoundProject();` has the same result, the test passes. The InitializeBoundProject is irrelevant in this case.